cfgov/apache/conf.d/alias.tmpl

```diff
@@ -0,0 +1,20 @@
+# These paths are served by WSGI as configured in wsgi.conf.
+# Bing site auth
+Alias /BingSiteAuth.xml ${STATIC_PATH}/BingSiteAuth.xml
+Alias /code.json ${STATIC_PATH}/code.json
+Alias /data.json ${STATIC_PATH}/data.json
+Alias /favicon.ico ${STATIC_PATH}/favicon.ico
+Alias /foia/quarterly ${STATIC_PATH}/foia/quarterly
+# Facebook site auth
+Alias /lln8595c61g9qnvuwvtlcwo1k6kem8.html ${STATIC_PATH}/lln8595c61g9qnvuwvtlcwo1k6kem8.html
+Alias /robots.txt ${STATIC_PATH}/robots.txt
+Alias /static/f/ ${STATIC_PATH}/nemo/_/f/
+# Preserves legacy URL compatibility, but this is not actually using Wordpress
+Alias /wp-content/themes/cfpb_nemo/_/  ${STATIC_PATH}/nemo/_/
+
+# Static assets served from files on disk.
```

Comment:
I think these staticsites sections should be removed now that #5092 moved rural-or-underserved-tool from a static site into Django.